lib/helpers.dart

```
@@ -97,6 +97,12 @@ Future<String?> getVNCPath() async {
     return join(dir.path, "rustdesk.exe");
   } else if (Platform.isMacOS) {
     return join(dir.path, "RustDesk.app");
+  } else if (Platform.isLinux) {
+    File rustdeskbinary = File("/usr/bin/rustdesk");
+    if (await rustdeskbinary.exists()) {
+      // assumes RustDesk has been installed via native Linux installer
+      return "/usr/bin/rustdesk";
+    }
```

Do we want to throw an error in the `else` case, instructing the user to install rustdesk?

---

Not necessary, because the getVNCPath() function will return null and then the robot.dart launchVNC() function will return with an error "Could not load RustDesk executable"

I tested the above by renaming /usr/bin/rustdesk

---

That's a good enough clue for a Linux user that they need to install rustdesk